app/helpers/application_helper.rb

```diff
@@ -31,6 +31,8 @@ def asset_exists?(asset_path)
   end
 
   def show_organisation_navigation_button?
+    return false if current_page?(organisations_path)
```

Comment:
On peut être plus général en faisant: 

```suggestion
    return false if if current_structure_type_in_params.blank?
```

Si on veut pousser on pourrait avoir une nouvelle méthode qui ne servirait qu'à checker s'il y une current structure dans les params (parce que normalement `current_structure_type_in_params` sert à renvoyer le type et pas faire ce check)